### ISSUE TITLE
ERROR: Automatic signature extraction failed: 

### ISSUE THREAD

**Issue by t0nym** [rg3/youtube-dl#3377]
This was working last week; just upgraded to youtube-dl_2014.06.19-1... and since to 2014.07.15-1 (as per research) - getting the above error. Even with URL's that worked back then. 
Further updated from 2014.06.19-1 as had "encryption" issues; research strongly suggested an update would correct issue. 

Error commentary: 
$ sudo youtube-dl http://www.youtube.com/watch?v=e6Qb-f_xpkI --verbose
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['http://www.youtube.com/watch?v=e6Qb-f_xpkI', '--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2014.07.15
[debug] Python version 2.7.7 - Linux-3.11-2-amd64-x86_64-with-SolydXK-1-testing
[debug] Proxy map: {}
[youtube] Setting language
[youtube] e6Qb-f_xpkI: Downloading webpage
[youtube] e6Qb-f_xpkI: Downloading video info webpage
[youtube] e6Qb-f_xpkI: Extracting video information
[youtube] e6Qb-f_xpkI: Encrypted signatures detected.
[youtube] encrypted signature length 84 (28.55), itag 43, html5 player en_US-vflCGk6yw\/html5player
ERROR: Automatic signature extraction failed: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 877, in _decrypt_signature
    video_id, player_url, len(s)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 352, in _extract_signature_function
    player_type = id_m.group('ext')
AttributeError: 'NoneType' object has no attribute 'group'
; please report this issue on https://yt-dl.org/bug . Be sure to call youtube-dl with the --verbose flag and include its complete output. Make sure you are using the latest version; type  youtube-dl -U  to update.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 877, in _decrypt_signature
    video_id, player_url, len(s)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 352, in _extract_signature_function
    player_type = id_m.group('ext')
AttributeError: 'NoneType' object has no attribute 'group'
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/YoutubeDL.py", line 516, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/common.py", line 168, in extract
    return self._real_extract(url)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 1249, in _real_extract
    encrypted_sig, video_id, player_url, age_gate)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 887, in _decrypt_signature
    u'Automatic signature extraction failed: ' + tb, cause=e)
ExtractorError: Automatic signature extraction failed: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 877, in _decrypt_signature
    video_id, player_url, len(s)
  File "/usr/lib/python2.7/dist-packages/youtube_dl/extractor/youtube.py", line 352, in _extract_signature_function
    player_type = id_m.group('ext')
AttributeError: 'NoneType' object has no attribute 'group'
; please report this issue on https://yt-dl.org/bug . Be sure to call youtube-dl with the --verbose flag and include its complete output. Make sure you are using the latest version; type  youtube-dl -U  to update.

Required details: 
$ youtube-dl --version
2014.07.15

$ python --version  (also just updated)
Python 2.7.7

$ uname -a
Linux <laptop_name> 3.11-2-amd64 #1 SMP Debian 3.11.10-1 (2013-12-04) x86_64 GNU/Linux

$ inxi -S
System:    Host: <laptop_name> Kernel: 3.11-2-amd64 x86_64 (64 bit) Desktop: KDE 4.12.4 Distro: SolydXK 1 testing

$ lsb_release
LSB Version:    core-2.0-amd64:core-2.0-noarch:core-3.0-amd64:core-3.0-noarch:core-3.1-amd64:core-3.1-noarch:core-3.2-amd64:core-3.2-noarch:core-4.0-amd64:core-4.0-noarch:core-4.1-amd64:core-4.1-noarch:security-4.0-amd64:security-4.0-noarch:security-4.1-amd64:security-4.1-noarch

$ cat /etc/debian_version 
jessie/sid

Basically, have updated a number of packages - not really sure this is the issue (except python, of course) - I'll be working to get back software versions as was in the meanwhile. 

Hopefully, the above is sufficient to assist in my enquiry. BTW - preceding with "sudo" also gives same response. 

Your assistance would be much appreciated on this issue - pls do not hesitate should you require any further information. 

TiA

t0nym
